pkg/controller/topology.go

```diff
@@ -108,7 +107,6 @@ func GenerateAccessibilityRequirements(
 	} else {
 		// selectedNode is set so use topology from that node to populate preferredTerms
 		// TODO (verult) reuse selected node info from aggregateTopologies
-		// TODO (verult) retry
```

Comment:
was retry logic added? or I assume there might be a higher level retry

Reply:
retry is done by the lib-external-provisioner